aliproxy/index.js

```
@@ -4,6 +4,39 @@ const { Buffer } = require('node:buffer')
+  'key',
+  'password',
+  'secret',
+  'session',
+  'signature',
+  'token',
+])
+
+function isSensitiveQueryKey(key) {
+  const lowerKey = key.toLowerCase()
+  const normalizedKey = lowerKey.replace(/[^a-z0-9]/g, '')
+  return SENSITIVE_QUERY_KEYS.has(lowerKey) || /(token|secret|password|apikey|auth|signature|session|code|key)/.test(normalizedKey)
```

ESLint

[error] Capturing group number 1 is defined but never used.

(regexp/no-unused-capturing-group)